buildconfig/setup_win_common.py

```diff
@@ -1,5 +1,3 @@
-# -*- encoding: utf-8 -*-
```

Comment:
Not sure about removing the encoding header. Removing is ok if the entire project uses utf-8 encoding only. On the other hand this might help an IDE to use the right encoding. 
I personally would leave it in and make sure each file has it. On the other hand this takes some maintenance effort.

Reply:
If your IDE thinks that the encoding for a Python 3 source file is anything other than UTF-8 then your IDE is mistaken.

Reply:
https://github.com/asottile/pyupgrade/blob/main/README.md#-coding--comment